Source/ASCollectionViewProtocols.h

```diff
@@ -10,6 +10,41 @@
 #import <UIKit/UIKit.h>
 #import <AsyncDisplayKit/ASBaseDefines.h>
 
+typedef NS_OPTIONS(NSUInteger, ASCellLayoutMode) {
+  /** No options set. Default value. */
+  ASCellLayoutModeNone = 0,
```

Comment:
Document the default behavior.

Reply:
Tried to clarify the default values for `ASCellLayoutMode` flags a bit better.